Stanford/JumpstartEngineering/Install/Courses/DisableCourses.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * @file
+ * Enable Courses
+ */
+
+namespace Stanford\JumpstartEngineering\Install\Courses;
+
+/**
+ *
+ */
+class DisableCourses extends \ITasks\AbstractInstallTask {
+
+  /**
+   * Disable Courses module on JSE sites
+   *
+   * @param array $args
+   *   Installation arguments.
+   */
+  public function execute(&$args = array()) {
+
+    module_load_include('module', 'redirect', 'redirect.module');
+    $modules = array( 'stanford_course_views', 'stanford_courses_administration',
+      'stanford_courses', 'stanford_feeds_helper');
+    module_disable($modules);
+
+    if (drupal_uninstall_modules($modules, FALSE)) {
+      drush_log('Disabled and uninstalled modules: ' . implode(', ', $modules), 'ok');
+    } else
+    {
+      drush_log('Error when uninstalling modules: ' . implode(', ', $modules), 'notice');
+    }
+
+    // Publish courses node
+    $query = new \EntityFieldQuery();
+
+    $entities = $query->entityCondition('entity_type', 'node')
+                      ->propertyCondition('type', 'stanford_page')
+                      ->propertyCondition('title', 'Courses Deprecated')
+                      ->execute();
+
+    if (!empty($entities['node'])) {
```

Comment:
This comment is purely about code style but it is a good discussion to have in a code review. I find having return statements much more clear when I am reading code instead of having to find the closing brace around a bunch of other code. I don't feel that they are random at all and shouldn't be missed by someone adding to the function later on. I would also argue that it would be better to create a new function rather than add to the current one. Keep the scope of each function to a sensible minimum. 

From the MIT coding standards guide:
> If you have a really long block, consider whether it can be broken into two or more shorter blocks or functions. If you consider such a long block unavoidable, put a short comment at the end on the same line the closing brace so people can tell at glance what that ending brace ends. Typically this is appropriate for a logic block, longer than about "35" rows, but any code that’s not intuitively obvious should be commented.

https://www.sitepoint.com/functional-programming-and-php/
https://en.wikipedia.org/wiki/Structured_programming

http://softwareengineering.stackexchange.com/questions/18454/should-i-return-from-a-function-early-or-use-an-if-statement